examples/using-react-ts/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
nit: I'd rather move this react example to its own demo repo rather than leaving it in the main repo here